### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by spuyet** [donnemartin/system-design-primer#250]
Maintainer(s): @spuyet

Please check out the [Translations Contributing Guidelines](https://github.com/donnemartin/system-design-primer/blob/master/CONTRIBUTING.md#translations).

Original translations thread: #28

Interested in helping? Let us know!

Translation branch: https://github.com/spuyet/system-design-primer/tree/add-french-translation

**Comment by donnemartin** [comment]
@spuyet awesome thank you!

**Comment by vnkomv** [comment]
hello @spuyet , i'm interested in helping with the translation.

**Comment by spuyet** [comment]
Hey @vnkomv, that's a good news :smiley:
I started to work on the french translation here: https://github.com/spuyet/system-design-primer/blob/add-french-translation/README-fr.md

Feel free to open PR, let me know if you want to focus on a specific part to avoid duplication of work ^^

**Comment by donnemartin** [comment]
Hi @spuyet: Are you still interested in translating the guide and being the language maintainer?

**Comment by spuyet** [comment]
Hello @donnemartin, yes I'm still interested, but doesn't have much time yet, if someone has more time to work on this you can give it to him :wink:

**Comment by pfongkye** [comment]
Hello @spuyet and @donnemartin,
I'm interested in contributing if it's still relevant.

**Comment by spuyet** [comment]
Hello @pfongkye, your contribution is welcome, I just merged master in: https://github.com/spuyet/system-design-primer/blob/add-french-translation/README-fr.md

You can make a PR at anytime :)

**Comment by AdilTestelin** [comment]
Hello @spuyet, 
Are you still interested about contributions ? 
Kind regards

**Comment by spuyet** [comment]
Hello @AdilTestelin, sorry for my late response. Every contribution is welcome !

**Comment by LaurelineP** [comment]
If I can help I would be happy to participate in the translations if you still need :) 
Best Regards :)

**Comment by spuyet** [comment]
Hello @LaurelineP, we're still looking for active contributors ! It would be a pleasure to review and merge one of your PR 👍

**Comment by maxx6262** [comment]
Hi there , i was founding about where I would apply to help you to translate into French language. 
Are you able to help me to understand how I would be more efficient to help team ?

**Comment by spuyet** [comment]
Hello @maxx6262, you just have to create a fork of https://github.com/spuyet/system-design-primer/tree/add-french-translation and create a PR from your branch to this one :wink:

**Comment by baptistemcn** [comment]
Hey guys, 
Still need help ?

**Comment by quichef** [comment]
Bonjour !
Will try to find some time to help here 😄 
Cheers

**Comment by Ptitet** [comment]
Hi @spuyet I'd like to help with translation, but I've seen that the status for the French translation is stalled, and there has been no activity since 2 years on your fork. Can I submit PRs to your repo ?

**Comment by quichef** [comment]
Yup pretty stall, @spuyet are you still around? Would it make sense for someone to help on that topic, or to retake ownership on the translation?

**Comment by spuyet** [comment]
@Ptitet @quichef You can send PR at anytime, they will be reviewed and merged 😉

**Comment by Venatum** [comment]
Hi 👋 

I’ve put together a pull request for this issue: [PR #7](https://github.com/spuyet/system-design-primer/pull/7)
It’s not fully polished yet (there are still a few tweaks to make), but I thought I’d share it in case anyone has feedback or suggestions

Thanks

**Comment by spuyet** [comment]
> Hello [@spuyet](https://github.com/spuyet), If you need help for the translation in french, I would like to contribute.

With pleasure !

**Comment by SouleymaneSy7** [comment]
Thank you for your positive response! I'm excited to contribute to this project. Could you please confirm if there are specific guidelines for the translation, such as:
- Is there a preferred tone or style for the translation (e.g., technical, formal, or casual)?
- Are there any other requirements or formatting rules I should follow?
Looking forward to your guidance so I can get started!

> > Hello [@spuyet](https://github.com/spuyet), If you need help for the translation in french, I would like to contribute.
> 
> With pleasure !
